deploy/hypershift-cluster-monitoring-config/config.yaml

```diff
@@ -2,7 +2,7 @@ deploymentMode: "SelectorSyncSet"
 selectorSyncSet:
   matchExpressions:
     - key: ext-hypershift.openshift.io/cluster-type
-      operator: In
+      operator: NotIn
```

Comment:
Doesn't this mean it would then land everywhere but management clusters - including standard osd/rosa clusters where it would clash with the other cluster-monitoring-config we push out?

Its content exactly matches our OSD/ROSA fleet config one: https://github.com/openshift/managed-cluster-config/blob/master/deploy/cluster-monitoring-config/config.yaml

Reply:
@supreeth7 We can remove the whole dir dirrectly since we are using FM to configure this.

Reply:
That's true 🤔 
Let me purge the entire directory.

Reply:
@mrbarge mind giving this another look? Thanks!